evm.md

```diff
@@ -1297,10 +1309,13 @@ The various `CALL*` (and other inter-contract control flow) operations will be d
          <gas> _ => GCALL </gas>
          <callGas> GCALL => 0 </callGas>
          <caller> _ => ACCTFROM </caller>
-         <program> _ => CODE </program>
-         <programBytes> _ => BYTES </programBytes>
+         <program> _ => .Map </program>
+         <programBytes> _ => .WordStack </programBytes>
          <static> OLDSTATIC:Bool => OLDSTATIC orBool STATIC </static>
          <touchedAccounts> ... .Set => SetItem(ACCTFROM) SetItem(ACCTTO) ... </touchedAccounts>
+         <activeAccounts> ACCTS </activeAccounts>
+         <schedule> SCHED </schedule>
+      requires notBool ACCTCODE in #precompiledAccounts(SCHED) andBool notBool ACCTCODE in ACCTS
```

Comment:
Put the `andBool` on the next line, and line up the `l` in `andBool` with the `s` in `requires`.